simulation/halsim_gui/src/main/native/cpp/HALSimGui.cpp

```diff
@@ -32,3 +32,23 @@ void HALSimGui::GlobalInit() {
 
   glass::AddStandardNetworkTablesViews(*ntProvider);
 }
+
+namespace halsimgui {
+
+void AddGuiInit(std::function<void()> initialize) {
+  wpi::gui::AddInit(initialize);
```

Comment:
These should use std::move to avoid an unnecessary copy.

Reply:
C++ is lame